cmudb/env/Dockerfile

```diff
@@ -1,23 +1,60 @@
 # NoisePage officially supports Ubuntu 20.04.
 FROM ubuntu:20.04
-# The default argument for docker run is bash.
-CMD ["bash"]
 
 # Suppress interactive dialog.
 ARG DEBIAN_FRONTEND=noninteractive
 
+# Copy the official PostgreSQL image for configurable env vars.
+# https://hub.docker.com/_/postgres/
+ENV POSTGRES_PASSWORD "terrier"
+ENV POSTGRES_USER "noisepage"
+ENV POSTGRES_DB "noisepage"
+ENV POSTGRES_INITDB_ARGS ""
+ENV POSTGRES_INITDB_WALDIR ""
+ENV POSTGRES_HOST_AUTH_METHOD ""
+
+# PostgreSQL env vars that we customize.
+ENV PGDATA "/pgdata"
+ENV PGPORT "15721"
+# NoisePage env vars.
+# BIN_DIR       :   Destination for PostgreSQL binaries.
+ENV BIN_DIR "/home/terrier/repo/build/bin"
+
 # Install packages.
-RUN apt-get -y update
-RUN apt-get -y install sudo
-COPY ./cmudb/env/packages.sh install-script.sh
-RUN echo y | ./install-script.sh all
-
-# Create a non-sudo user.
-RUN adduser --disabled-password --gecos '' dev \
-    && adduser dev sudo \
+COPY ./cmudb/env/packages.sh /packages.sh
+RUN \
+    apt-get -y update \
+    && apt-get -y install sudo \
+    && echo y | /packages.sh all
```

Comment:
I'm not sure if I was doing something wrong, but whenever I connected to one of the containers, I had to manually install the postgres client. It might be nice to just install it here to avoid that.

Reply:
I'm just realizing that I'm pretty sure I could have just ran psql from my host machine without needing to run a terminal in the container. If that's the case then just ignore these comments.

Reply:
Yes, we should be able to run psql from the host machine. There should also be a psql binary in the `bin` folder of the container as part of `make install`.